src/js/api.ts

```diff
@@ -520,7 +520,7 @@ export class PyodideAPI {
       buffer,
       format,
       extract_dir,
-      installer: "pyodide.unpackArchive",
+      metadata: new Map([["INSTALLER", "pyodide.unpackArchive"]]),
```

Comment:
Rather than making a new map every time maybe we could make this a top level constant?

Reply:
I guess this isn't used that much.

Reply:
Thanks. I think it is good to make it a constant.